### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Concelare**
Update to README.md, Adding Build/Usage instructions as well as configuration setup

### CODE REVIEW

**Comment by extrawurst** [comment]
lets put that in a `CONTRIBUTION.md` and link it only from the readme

File: README.md [hunk 1]
```diff
@@ -37,7 +37,58 @@ Live-Ask is a product developed and maintained by [Rustunit.com](https://rustuni
 See detailed [changelog](CHANGELOG.md).
 
 ## Build & Usages Details
-(coming soon)
+**Requires Three Terminal Tabs/Instances**
+### Initial Setup
+#### Pre-requisites 
+```
+rustup update
```
Comment (extrawurst) [inline README.md:44]: maybe also a link to a `how to install rust` source

File: README.md [hunk 2]
```diff
@@ -37,7 +37,58 @@ Live-Ask is a product developed and maintained by [Rustunit.com](https://rustuni
 See detailed [changelog](CHANGELOG.md).
 
 ## Build & Usages Details
-(coming soon)
+**Requires Three Terminal Tabs/Instances**
+### Initial Setup
+#### Pre-requisites
```
Comment (extrawurst) [inline README.md:42]: list all prerequesits, like `docker`

File: README.md (hunk 1 above)
Comment (Concelare) [reply README.md:44]: Will do now

**Comment by Concelare** [comment]
> lets put that in a `CONTRIBUTION.md` and link it only from the readme

I have done this 👍

**Comment by extrawurst** [comment]
good job!

**Pull request merged**